app-admin/logstash-bin/files/logstash-plugin.eselect

```diff
@@ -0,0 +1,80 @@
+# -*-eselect-*-  vim: ft=eselect
+# Copyright 2005-2017 Gentoo Foundation
+# Distributed under the terms of the GNU GPL version 2 or later
+
+DESCRIPTION="Manage user installed plugins of logstash"
+MAINTAINER="hydrapolic@gmail.com"
+
+PLUGINS="/var/lib/logstash/plugins-list.txt"
+LS_INSTALL_DIR="/opt/logstash"
+
+ls_plugin_wrapper() {
+  cd "${LS_INSTALL_DIR}" || die
+  JARS_SKIP=true ./bin/logstash-plugin "$1" "$2"
```

Comment:
Seems like this was the only way to run it in the last IIRC, but you are right, now it works even running with full path.